rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date
```

Comment:
- [ ] Please also add `:book` and `:person` to `:date` because they all require getters and setters.